PyInstaller/hooks/hook-gi.repository.GdkPixbuf.py

```diff
@@ -33,14 +33,13 @@
         binaries.append((f, 'lib/gdk-pixbuf-2.0/2.10.0/loaders'))
     
     # Create an updated version of the loader cache
-    cachedata = subprocess.check_output('gdk-pixbuf-query-loaders')
+    cachedata = subprocess.check_output('gdk-pixbuf-query-loaders', universal_newlines=True)
```

Comment:
This is to provide python 3 compatibility. The universal_newline causes check_output to return a string instead of bytes.

After looking at this again I no longer think this is the right thing to do.